internal/command/jobrunnergroup/printers.go

```diff
@@ -0,0 +1,61 @@
+package jobrunnergroup
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/signadot/cli/internal/utils"
+	"github.com/xeonx/timeago"
+
+	"text/tabwriter"
+
+	"github.com/signadot/cli/internal/config"
+	"github.com/signadot/cli/internal/sdtab"
+	"github.com/signadot/go-sdk/models"
+)
+
+type runnerGroupRow struct {
+	Name    string `sdtab:"NAME"`
+	Cluster string `sdtab:"CLUSTER"`
+	Created string `sdtab:"CREATED"`
+	Status  string `sdtab:"STATUS"`
+}
+
+func printRunnerGroupTable(cfg *config.JobRunnerGroupList, out io.Writer, rgs []*models.RunnergroupsRunnerGroup) error {
+	t := sdtab.New[runnerGroupRow](out)
+	t.AddHeader()
+	for _, rg := range rgs {
+		createdAt, err := time.Parse(time.RFC3339, rg.CreatedAt)
+		if err != nil {
+			return err
+		}
+
+		t.AddRow(runnerGroupRow{
+			Name:    rg.Name,
+			Cluster: rg.Spec.Cluster,
+			Created: timeago.NoMax(timeago.English).Format(createdAt),
+			Status:  readiness(rg.Status),
+		})
+	}
+	return t.Flush()
+}
+
+func printRunnerGroupDetails(cfg *config.JobRunnerGroup, out io.Writer, rg *models.RunnergroupsRunnerGroup) error {
+	tw := tabwriter.NewWriter(out, 0, 0, 3, ' ', 0)
+
+	fmt.Fprintf(tw, "Name:\t%s\n", rg.Name)
+	fmt.Fprintf(tw, "Created:\t%s\n", utils.FormatTimestamp(rg.CreatedAt))
+	fmt.Fprintf(tw, "Status:\t%s\n", readiness(rg.Status))
+	fmt.Fprintf(tw, "Dashboard page:\t%s\n", cfg.RunnerGroupDashboardUrl(rg.Name))
+
+	if err := tw.Flush(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func readiness(status *models.RunnergroupsStatus) string {
+	return fmt.Sprintf("%d/%d pods ready", status.Pods.Ready, status.Pods.Ready+status.Pods.NotReady)
```

Comment:
Looks like we'll need to account for idle as well.
<img width="187" alt="image" src="https://github.com/signadot/cli/assets/3791487/c7fe5e0e-9ef9-4c2b-9efa-6803018dc824">

Alternately, we should surface more details - including the counts for total, ready, not ready and idle. Sorry I hadn't accounted for this in the UX as the statuses were not clear at that point.

Reply:
I also had Idle but @foxish make some changes

Reply:
I'd ignore idle, and just have ready / not-ready because those two are collectively exhaustive. Idle / not is a separate field and it's ok to not have that info right now.

Reply:
ack